.vscode/launch.json

```diff
@@ -4,6 +4,13 @@
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
     "version": "0.2.0",
     "configurations": [
+        {
```

Comment:
Putting the comment here because I can't put it on the `.DS_Store` files (there are multiple). I would delete these and add them to the `.gitignore`. These are generated in VS Code and don't need to be committed to the repository.

Reply:
done

Reply:
I added a new comment about the `.DS_Store` files. I see you got rid of the `.vscode/launch.json` where I started this thread; which is good. I'm going to approve, but I would recommend getting rid of these soon.